src/routes/MapExplorer/MapDrawer/MapDrawer.js

```diff
@@ -43,121 +52,165 @@ function MapDrawer (props) {
+                onClick={removeTopicFilter(topic)}
                 styleName='topicButton'
               >
-                <span styleName='topicCount'>{topic.count}</span> {topic.name}
+                <span styleName='topicCount'>{topic.count}</span> #{topic.name} <Icon name='Ex' className={styles.filterEx} />
               </span>
             )
           })}
         </div>
-        : ''
-      }
 
-      <div styleName='currentFilters'>
-        {searchText
-          ? <div
-            styleName='currentSearchText'
-            onClick={() => onUpdateFilters({ search: '' })}
-          >
-            &quot;{searchText}&quot; <Icon name='Ex' className={styles.textEx} />
-          </div>
-          : ''
-        }
-        {filters.topics.map(topic => {
-          return (
-            <span
-              key={'filter_topic_' + topic.name}
-              onClick={removeTopicFilter(topic)}
-              styleName='topicButton'
-            >
-              <span styleName='topicCount'>{topic.count}</span> #{topic.name} <Icon name='Ex' className={styles.filterEx} />
-            </span>
-          )
-        })}
+        <TabBar currentTab={currentTab} tabs={tabs} selectTab={setCurrentTab} pendingPostsDrawer={pendingPostsDrawer} />
       </div>
 
-      <h1>{features.length} result{features.length === 1 ? '' : 's'} in this area { pending && <Loading type='inline' styleName='loading' /> }</h1>
-      <Dropdown styleName='sorter'
-        toggleChildren={<span styleName='sorter-label'>
-          {SORT_OPTIONS.find(o => o.id === sortBy).label}
-          <Icon name='ArrowDown' className={styles.sorterIcon} />
-        </span>}
-        items={SORT_OPTIONS.map(({ id, label }) => ({
-          label,
-          onClick: () => onUpdateFilters({ sortBy: id })
-        }))}
-        alignRight
-      />
-
-      <div styleName='contentListContainer'>
-        {featuresHTML}
+      {currentTab === 'Posts' ? <div styleName='contentWrapper'>
+        <div styleName='postsHeader'>
+          <span>Sort posts by:</span>
+          <Dropdown styleName='sorter'
+            toggleChildren={<span styleName='sorter-label'>
+              {SORT_OPTIONS.find(o => o.id === sortBy).label}
+              <Icon name='ArrowDown' className={styles.sorterIcon} />
+            </span>}
+            items={SORT_OPTIONS.map(({ id, label }) => ({
+              label,
+              onClick: () => onUpdateFilters({ sortBy: id })
+            }))}
+            alignRight
+          />
+        </div>
+
+        <div styleName='contentListContainer' id='contentList'>
+          {posts.map(p => <PostCard
+            routeParams={routeParams}
+            post={p}
+            styleName='contentCard'
+            constrained
+            expanded={false}
+            key={p.id}
+          />)}
+        </div>
+
+        <ScrollListener onBottom={() => fetchPostsForDrawer(numFetchedPosts, false)} elementId='mapDrawerWrapper' />
```

Comment:
Does this one also fetch groups then now?

Reply:
or wait, I think you send you just request all public groups

Reply:
fetchGroups is its own query https://github.com/Hylozoic/hylo-evo/blob/616-map-load-performance/src/routes/MapExplorer/MapExplorer.store.js

yeah i just fetch all public groups in a bounding box for now